pokemongo_bot/cell_workers/pokemon_catch_worker.py

```diff
@@ -44,6 +44,7 @@
     'andor': lambda x, y, z: x and y or z
 }
 
+DEBUG_ON = False
 
```

Comment:
this flag only applies to this rather than using global debug which generates a lot of garbage